plugins-compat-tester/src/main/java/org/jenkins/tools/test/model/PluginRemoting.java

```diff
@@ -213,12 +207,6 @@ public static void computeScmConnection(PomData pomData) {
             pomData.getWarningMessages().add("project.scm.connectionUrl should be ending with '-plugin.git'");
         }
 
-        oldUrl = transformedConnectionUrl;
-        transformedConnectionUrl = transformedConnectionUrl.replace("git.cloudbees.com", "github.com");
-        if (!oldUrl.equals(transformedConnectionUrl)) {
-            pomData.getWarningMessages().add("project.scm.connectionUrl was using git.cloudbees.com; moved to GitHub");
-        }
-
```

Comment:
IMHO, this should never have been here.

Reply:
This was actually necessary at one point, probably before your time, but is obsolete now.